scripts/skip-test-if.js

```diff
@@ -1,56 +0,0 @@
-/*
```

Comment:
please keep this - we'll likely need it for other instrumentations going forward.